packages/snaps-rpc-methods/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.5.3]
+
+### Fixed
+
+- Wait for unlock before handling `manageAccounts` requests ([#3686](https://github.com/MetaMask/snaps/pull/3686))
+- Throw if Snap not installed ([#3666](https://github.com/MetaMask/snaps/pull/3666))
```

Comment:
Since this removes a throw in certain cases we need to release snap-controllers too